### PR TITLE
Vine: Add IO test for function calls

### PR DESCRIPTION
**Pull request by tphung3**
## Proposed Changes

This PR adds an IO test (input/output staging) for function calls.

## Merge Checklist

The following items must be completed before PRs can be merged.
Check these off to verify you have completed all steps.

- [x] `make test`       Run local tests prior to pushing.
- [x] `make format`     Format source code to comply with lint policies. Note that some lint errors can only be resolved manually (e.g., Python)
- [x] `make lint`       Run lint on source code prior to pushing.
- [ ] Manual Update:     Update the manual to reflect user-visible changes.
- [x] Type Labels:       Select a github label for the type: bugfix, enhancement, etc.
- [x] Product Labels:    Select a github label for the product: TaskVine, Makeflow, etc.
- [x] PR RTM:            Mark your PR as ready to merge.

### CODE REVIEW

**Comment by tphung3** [comment]
@btovar I added the IO test for function calls here and they seem to work as intended. Maybe we need a test where a function call should fail when a file isn't brought back?

File: taskvine/test/TR_vine_python_serverless.sh [hunk 1]
```diff
@@ -11,6 +11,9 @@ export PYTHONPATH=$(pwd)/../../test_support/python_modules/${CCTOOLS_PYTHON_TEST
 
 STATUS_FILE=vine.status
 PORT_FILE=vine.port
+TEST_FILE=vine_python_serverless.py
```
Comment (btovar) [inline taskvine/test/TR_vine_python_serverless.sh:14]: I'd recommend against naming this variable `TEST_FILE`. It gives the impression that the file can be removed. Use PYTHON_SCRIPT?

**Review by btovar — CHANGES_REQUESTED** [review]

**Review by btovar — APPROVED** [review]

**Comment by btovar** [comment]
ready to merge?

**Comment by tphung3** [comment]
It's RTM.

**Pull request merged**